### ISSUE TITLE
Indent: tabs vs spaces

### ISSUE THREAD

**Issue by ericpoe** [phpbridge/docs#38]
Files are a mix of tabs and spaces for indents. Which should we use? If spaces, how many?

**Comment by abelardolg** [comment]
4 Spaces

**Comment by ericpoe** [comment]
@abelardolg, yeah, 3 years later, I'd say [PSR-2](http://www.php-fig.org/psr/psr-2/#overview) all the way!
